docs/framework/migration-guide/reference-assemblies.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Build apps against Microsoft.NETFramework.ReferenceAssemblies"
+description: "Developers can target .NET Framework by referencing the official reference assemblies NuGet package."
+ms.date: 11/25/2022
+helpviewer_keywords:
+  - ".NET Framework, targeting"
+---
+# Build apps against Microsoft.NETFramework.ReferenceAssemblies
+
+When you target a particular version of .NET Framework in a traditional, non-SDK-style project, by default your application is built by using the reference assemblies that are included with that version's developer pack. In scenarios where the matching developer pack cannot be installed on the computer, you can alternatively build against reference assemblies distributed via a NuGet package.
+
+## Update project files
+
+Each project that should build against the reference assemblies NuGet package needs to include the following package reference.
+
+```xml
+<ItemGroup>
+  <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.3" PrivateAssets="All" />
+</ItemGroup>
+```
+
+This can be done with the NuGet Package Manager UI in Visual Studio, with [dotnet add package](../../core/tools/dotnet-add-package.md) on the .NET SDK command line, or by manually editing the project file. The package contains reference assemblies for many versions of .NET Framework. The version to be actually used is determined by the TargetFramework (TargetFrameworks) property.
+
+> [!NOTE]
+> Projects that contain a project reference need to be restored before they can be built. When building with Visual Studio, either in the IDE or on the command line with *MSBuild.exe*, make sure to explicitly execute the *restore* target prior to building the project. This is done on the Developer Command Prompt by executing `msbuild /t:restore` in the project directory.
```

Comment:
Also, running `msbuild /t:restore` is really something someone would run if they were using Framework-style projects and not SDK-style projects. So along with my other suggestion of including instructions for both project styles, I would update this specifically to be `dotnet restore` and then use `msbuild` in the Framework-style section.

Reply:
This guidance is primarily for non-SDK-style ("Framework-style" as you wrote) projects. SDK-style works out of the box because they already have the `PackageReference` included from the .NET SDK.

SDK-style still needs to run restore, but that's something they have to do anyways, so it looks as though including it here would be redundant. Although on second thought, it makes sense that SDK-style devs would land here too, and the step to restore may not be obvious, especially since "Restore NuGet Packages" in VS doesn't cut it.

OK, let me restructure it a bit. Thank you!

Reply:
I ran through the scenarios again and have updated the text.
- The build prefers the machine-wide targeting pack in all cases. So I have removed the "non-SDK-style" part from the first paragraph. The new text is more correct and concise.
- The section about updating project files explains the difference between SDK and non-SDK. It doesn't recommend `dotnet add-package` anymore, because it's not needed for SDK and doesn't work for non-SDK.
- A new section explains how to restore on the command line. The difference here is not between SDK and non-SDK, but between VS and dotnet. While likely equivalent in most cases, I think it's better to recommend `msbuild /t:restore` even for SDK-style projects if the developer is in VS. @zivkan, could you please comment on this? Thank you!

Reply:
Some (most? all?) .NET Framework (non-SDK) projects can't be built, or restored, by the `dotnet` CLI. The project files import props/targets that only ship in VS. However, VS and msbuild is able to discover the .NET SDK. So, `msbuild -t:restore` will work for anyone with VS installed, but `dotnet restore` will only work for solutions where every* project in the solution is SDK style. Just one non-SDK style project (like a Visual Studio Extension VSIX project) will cause `dotnet` to fail MSBuild evaluation.

So, while I personally do recommend customers with all SDK style projects in their solution to use the dotnet cli (just so the skills are transferrable to Linux & Mac, no technical reason), it is safe to tell anyone on Windows with VS installed to use msbuild.

*I think that very basic console and classlib projects might work with the dotnet cli, but it's just easier to say something like "non-SDK style project, you must use msbuild".

Reply:
Thank you. The text now says "When building with VS ... use `nuget restore` or `msbuild /t:restore`" and "When building with .NET SDK ... use `dotnet restore`"). That should be safe / work for everyone, unless I misunderstood.